stat/stat_test.go

```diff
@@ -74,9 +74,6 @@ func TestLstat(t *testing.T) {
 				s = WithTimeout(1*time.Nanosecond, attempts, consecutiveFails, l)
 				So(s, ShouldNotBeNil)
 
-				defer func() { os.Unsetenv("WRSTAT_TEST_LSTAT") }()
-				os.Setenv("WRSTAT_TEST_LSTAT", "long")
-
 				info, err = s.Lstat(pathContent1)
 				So(err, ShouldNotBeNil)
```

Comment:
How are these tests working/useful without the WRSTAT_TEST_LSTAT making the stat take a long time?

Reply:
No idea, but I was initially going to repalce the env var with a global bool that the test would set, but the tests still succeeded without the bool being set in the test, so I removed the code assumming that it was just old code.